.github/workflows/release.yml

```diff
@@ -8,26 +8,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: '3.x'
       - uses: actions/setup-java@v1
         with:
           java-version: 11
-      - run: pip3 install -r requirements.txt
-      - run: ./build.sh
       - run: ./gradlew build
         env:
+          RELEASE_NOTES: ${{ github.event.release.body }}
           VERSION: ${{ github.ref }}
-      - name: Copy jar file
-        run: cp build/libs/*.jar one-dark-theme.jar
+      - name: Copy zip file
+        run: cp build/libs/*.zip one-dark-theme.zip
```

Comment:
The plugin has bundled dependencies now, so the original (one dark) jar is now packaged with it's other jar buddies so the plugin can work.